src/mockServiceWorker.js

```diff
@@ -147,20 +147,22 @@ async function handleRequest(event, requestId) {
   if (client && activeClientIds.has(client.id)) {
     ;(async function () {
       const clonedResponse = response.clone()
-      sendToClient(client, {
+      const body =
```

Comment:
Let's read the request body as text for now. Removing the payload stringification and changing the request body type are two separate things, we can ship them one by one but isolated.